x-pack/test/screenshot_creation/apps/ml_docs/anomaly_detection/influencers.ts

```diff
@@ -67,11 +64,6 @@ export default function ({ getService }: FtrProviderContext) {
       await ml.jobTable.filterWithSearchString(populationJobConfig.job_id, 1);
       await ml.jobTable.clickCloneJobAction(populationJobConfig.job_id);
       await ml.jobTypeSelection.assertPopulationJobWizardOpen();
-
-      await ml.testExecution.logTestStep('continue to the pick fields step and take screenshot');
-      await ml.jobWizardCommon.advanceToPickFieldsSection();
-      await commonScreenshots.removeFocusFromElement();
-      await commonScreenshots.takeScreenshot('ml-population-job', screenshotDirectories);
```

Comment:
Since we're not taking this screenshot anymore, I think we can remove the whole `it('wizard screenshot', ...` test block.